test/functional/feature_utxo_set_hash.py

```diff
@@ -3,19 +3,24 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test UTXO set hash value calculation in gettxoutsetinfo."""
-
-import struct
-
 from test_framework.messages import (
-    CBlock,
+    COIN,
     COutPoint,
-    from_hex,
+    CTxOut,
 )
 from test_framework.muhash import MuHash3072
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import assert_equal
 from test_framework.wallet import MiniWallet
 
+
+def txout_serialize(txid_hex, n, height, is_coinbase, value, scriptPubKey_hex):
```

Comment:
Okay, I guess you just used snake case of the `TxOutSer` function name. On that note, I think it would be good to explicitly reference `TxOutSer` (name is fine imo) here so it's easier for people where this logic is coming from?

Reply:
Good ideas, applied both the type hints and the s/n/output_index rename. Also add a small comment referring to the original serialization function `TxOutSer`.